tests/components/zwave_js/test_api.py

```diff
@@ -3299,7 +3299,7 @@ async def test_subscribe_log_updates(
 
     # Test FailedZWaveCommand is caught
     with patch(
-        "zwave_js_server.model.driver.Driver.async_start_listening_logs",
+        "zwave_js_server.client.Client.async_start_listening_logs",
```

Comment:
We've already patched the client in the client fixture. Just access the client fixture and set the side_effect on the client attribute directly.